### DISCUSSION TITLE
Docker Image not working

### DISCUSSION THREAD

**Discussion by ndtrung00** [fuzzland/ityfuzz#252] [Q&A] [answered]
Hi team, 

I am currently doing my Bachelor's thesis and have figured out the tool. Great work! I am trying to understand the tool better.

During the process, i noticed that the docker image is currently not working. It worked before but seems like your API key to etherscan is expired. 

I tried to start fuzzing from the UI and it crashes every time. 

I built the tool on my local machine and ran it with the command you guys provided in the Readme.md also doesn't work and its bc of the API Key to etherscan. I changed to mine and it worked again.

Is it possible that you guys somehow update the image to execute the fuzz with the working API Key so that i can use the UI again?

Thank you guys so much for your time!

**Comment by 0xAWM** [comment]
Thanks for ur reporting. I'll work on it. Maybe this weekend or later

**Reply by ndtrung00** [reply]
thank you so much for the response. Should i help you create an issue?

**Reply by 0xAWM** [reply, marked as answer]
@TrungNguyen1409  Thx. 
After #255  CI passes, you can set your own etherscankey through ENV.
